core/src/main/java/org/neo4j/ogm/metadata/ClassInfo.java

```diff
@@ -716,11 +714,11 @@ public Method getMethod(MethodInfo methodInfo) {
                     && relationshipDirection.equals(Relationship.INCOMING))
                     || (relationshipDirection.equals(Relationship.OUTGOING) && !(fieldInfo
                     .relationshipDirection(Relationship.OUTGOING).equals(Relationship.INCOMING)))) {
-                    fieldInfos.add(fieldInfo);
+                    iterableFields.add(fieldInfo);
```

Comment:
Just regarding code style: Why is there no streaming api usage as e.g. in the change above? (With extraction of filter predicate)